### ISSUE TITLE
Support for editing code on mobile. (Use react-live!)

### ISSUE THREAD

**Issue by trusktr** [SoftwareBrothers/better-docs#18]
Hello! If you're using React, you should totally use [react-live](https://github.com/FormidableLabs/react-live)'s editor component. It works on mobile, because it uses contenteditable for its editing abilities.

At the moment, I have trouble with the current editor in your docs on my phone (editors like Monaco, CodeMirror, and Ace, don't work on mobile).

**Comment by wojtek-krysiak** [comment]
good idea, definitely worth doing this, but this is not an easy task so I cannot give you any deadlines for that
